test/hotspot/gtest/metaspace/test_virtualspacenode.cpp

```diff
@@ -540,8 +540,8 @@ TEST_VM(metaspace, virtual_space_node_test_basics) {
   DEBUG_ONLY(node->verify_locked();)
 
   delete node;
-  ASSERT_EQ(scomm.get(), 0);
-  ASSERT_EQ(sres.get(), 0);
+  ASSERT_EQ(scomm.get(), size_t{0});
+  ASSERT_EQ(sres.get(), size_t{0});
```

Comment:
While I personally like this, could you please keep cast style as in the rest of the hotspot? C-style or xxx_cast, your call.